sky/clouds/service_catalog/gcp_catalog.py

```diff
@@ -116,6 +116,7 @@
     for instance_type in instance_types
 }
 _ACC_INSTANCE_TYPES = list(_INSTANCE_TYPE_TO_ACC.keys())
+GCP_ACC_INSTANCE_TYPES = _ACC_INSTANCE_TYPES
```

Comment:
Should we just use the same variable name?
```suggestion
GCP_ACC_INSTANCE_TYPES = list(_INSTANCE_TYPE_TO_ACC.keys())
```